StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1611ElementParametersMustBeDocumented.cs

```diff
@@ -19,6 +19,7 @@
     /// more of its parameters.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce warning SA1614, but at least the user wouldn't have to type the XML portion manually.

```csharp
/// <param name="S"></param>
```